lib/dotcom_web/components/planned_disruptions.ex

```diff
@@ -80,7 +82,11 @@ defmodule DotcomWeb.Components.PlannedDisruptions do
 
   # Extracts the start and stop times from the active period of an alert.
   defp alert_date_time_range(%Alert{active_period: active_period}) do
-    periods = Enum.sort_by(active_period, fn {start, _} -> start end)
+    periods = Enum.sort_by(
```

Comment:
Is this different? Do datetime's get sorted by default by some function other than `DateTime.compare/2`?

If so, can we add a test that fails when this uses default sorting?

Also - non-blocking, but I find
```elixir
fn {start, _} -> start end
```

easier to read than
```elixir
&Kernel.elem(&1, 0)
```

(and relatedly - if you do want to keep the `elem` invocation, I thought it was possible to just call `elem` without invoking it as `Kernel.elem`?)